src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -21,6 +24,30 @@ function ManufacturerLandingPage() {
   const [editManufacturerDialogOpen, setEditManufacturerDialogOpen] =
     React.useState<boolean>(false);
 
+  const navigate = useNavigate();
+  const onChangeNode = React.useCallback(
+    (id: string | null) => {
+      navigate(id ? `${paths.manufacturers}/id` : paths.manufacturers);
```

Comment:
```suggestion
      navigate(id ? `${paths.manufacturers}/${id}` : paths.manufacturers);
```
Not that it is called anyway